README.md

```diff
@@ -84,6 +112,11 @@ More instructions for setup can be found in the [Elastic Stack Installation Guid
 
 #### fleet-server stand alone
 
+Fleet UI requires a managed Fleet Server, to be able to use stand alone Fleet
+server, you need to enroll a managed Fleet Server or disable this requirement.
+You can disable this requirement since Kibana 8.8.0, starting it with
+`xpack.fleet.enableExperimental: ['fleetServerStandalone']`.
```

Comment:
```suggestion
Fleet UI requires a managed Fleet Server, to be able to use stand alone Fleet
server, you need to enroll a managed Fleet Server or disable this requirement.
You can disable this requirement since Kibana 8.8.0, starting it with
`xpack.fleet.enableExperimental: ['fleetServerStandalone']`. This is only
supported internally and is not intended for end-users at this time.
```